src/auto-redis/autocomplete.ts

```diff
@@ -5,19 +5,23 @@ export default function autocomplete ({pzrank, pzrange, zset}: {pzrank:any, pzra
 
     const rank = await pzrank(zset, payload);
     const inc = 128;
-
+    console.log(rank);
     async function internal(init: number, end: number, acc: string[] = []): Promise<string[]> {
-      const nextPage = await pzrange(zset, init, end);
-      const results = [
-        ...acc,
-        ...nextPage.filter((gd: string) => gd.startsWith(payload) && gd.charAt(gd.length -1) === '*')
-                  .map((gd: string) => gd.substring(0, gd.length -1))
-      ];
-      const last = nextPage.pop();
-      if (last && last.startsWith(payload)) {
-        return internal(init + inc, end + inc, results);
-      } else {
-        return results;
+      if(init) {
```

Comment:
I think init is a misleading name ([init](https://en.wikipedia.org/wiki/Init)) i would call it `from`